nitrite_test.go

```diff
@@ -7,79 +7,141 @@ import (
 	"testing"
 	"time"
 
-	"github.com/blocky/nitrite"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/blocky/nitrite"
 )
 
-// An easy way to create a new nitro attestation is to use delphi-cli.
-// Request an attestation over some endpoint and copy the resulting
-// "enclave_attestation" string from the output---it should be a
-// base64 encoded string
-//
-//go:embed testdata/nitro_attestation.txt
-var nitroAttestString string
+// The canonical way to regenerate attestations is to request an attestation
+// using hf/nsm/Send() and write the resulting bytes to a file as a base64
+// string.
```

Comment:
It would be nice if there was a little more tooling in place to be able to regenerate the test data other than pointing to a function in a library, but maybe that is out of scope for this PR.

Reply:
Yeah, updates are very infrequent though. I used some on the fly changes to the `knight` `nitro_nsm_attest` test to save the attestations after tweaking enclave startup mode with and without debug. Pretty easy, if a little manual.

For the self-signed attestation grabbing it shouldn't be too hard. 

I think we could add a card to do something more automated using knight, or delphi, once we have Google Confidential Spaces integrated and some additional code (not in this repo) to verify those attestations. At that point we'll probably remove the self-signed stuff too.

I'd make a card, but I think that its going to come back anyway and we'll be in a better position to do something automated when it does come back around.